### PR TITLE
Fixed decompiler for osx

### PR DESCRIPTION
**Pull request by cuff-links**
Made some edits to make it easier to set up the template project on OSX.

### CODE REVIEW

**Comment by cuff-links** [comment]
@ColonolNutty ^

File: decompile_all.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def unpyc3(include_ea_decompileile=False, include_decompile_directory=False):
         decompile_dir(decompile_src)
     if include_ea_decompileile:
         gameplay_folder_data = os.path.join(game_folder, 'Data', 'Simulation', 'Gameplay')
-        gameplay_folder_game = os.path.join(game_folder, 'Game', 'Bin', 'Python')
+        gameplay_folder_game = os.path.join(game_folder, 'Python')
```
Comment (ColonolNutty) [inline decompile_all.py:16]: It still needs to work for Windows. I'd recommend we put an if check here that checks if it is on a Mac or not.

File: settings.py [hunk 2]
```diff
@@ -22,7 +29,7 @@
 # 4. Inside of the <Project>/EA folder, you should see four folders (base, core, generated, simulation)
 # 5. Highlight all four of those folders and right click them. Then do Mark Directory as... Sources Root
 # 6. Delete the <Project>/EA/core/enum.py file because it causes issues when attempting to compile the scripts of your own mod.
-include_ea_decompile = False
+include_ea_decompile = True
```
Comment (ColonolNutty) [inline settings.py:32]: You should change this back to `False` so as not to change the behavior of the defaults.

File: settings.py [hunk 3]
```diff
@@ -7,9 +7,16 @@
...
-mods_folder = os.path.expanduser(os.path.join('~', 'Documents', 'Electronic Arts', 'The Sims 4', 'Mods'))
+if os.name == 'posix':
+    os_home_prefix = os.environ['HOME']
+else:
+    os_home_prefix = '~'
+mods_folder = os.path.join(os_home_prefix, 'Documents', 'Electronic Arts', 'The Sims 4', 'Mods')
 # Change to the location where you have installed The Sims 4 at, this would be the folder that contains the GameVersion.txt file
-game_folder = os.path.join('E:', os.sep, 'Program Files (x86)', 'Origin Games', 'The Sims 4')
+if os.name == 'posix':
+    game_folder = os.path.join(os.environ['HOME'], 'Applications', 'The Sims 4.app', 'Contents')
```
Comment (ColonolNutty) [inline settings.py:17]: You should use the `os_home_prefix` variable from above, instead of `os.environ['HOME']`.

File: decompile_all.py (hunk 1 above)
Comment (cuff-links) [reply decompile_all.py:16]: Thanks. I was hoping I didn't miss any. :)

File: settings.py (hunk 3 above)
Comment (cuff-links) [reply settings.py:17]: Fixed.

File: decompile_all.py (hunk 1 above)
Comment (cuff-links) [reply decompile_all.py:16]: Fixed.

File: settings.py [hunk 4]
```diff
@@ -4,12 +4,19 @@
 # Example:
 # If I set the value to "ColonolNutty"
 # compiling my scripts would output a file with the name "ColonolNutty_<script_name>.ts4script"
-creator_name = ''
+creator_name = 'Cufflinks'
```
Comment (ColonolNutty) [inline settings.py:7]: You probably don't want this.

File: settings.py (hunk 2 above)
Comment (ColonolNutty) [reply settings.py:32]: This still needs to be fixed.

File: settings.py (hunk 4 above)
Comment (cuff-links) [reply settings.py]: Fixed.

**Pull request merged**